### PR TITLE
Small Changes to motor documentation, matched changes with vision

### PR DESCRIPTION
**Pull request by joebuss**

### CODE REVIEW

File: v5/api/c/motors.rst [hunk 1]
```diff
@@ -116,7 +116,7 @@ Analogous to `pros::Motor::move_absolute <../cpp/motors.html#move-absolute>`_.
 ============ ===============================================================
  port         The V5 port number from 1-21
  position     The absolute position to move to in the motor's encoder units
- velocity     The maximum allowable velocity for the movement
+ velocity     The maximum allowable velocity for the movement in RPM
```
Comment (Octogonapus) [inline v5/api/c/motors.rst]: Is this really RPM or is it the motor's encoder units?

File: v5/api/c/vision.rst [hunk 2]
```diff
@@ -972,7 +967,7 @@ This function uses the following values of errno when an error state is
 reached:
 
 - ``ENXIO`` - The given value is not within the range of V5 ports (1-21).
-- ``ENODEV``  - The device cannot be configured as a vision sensor (e.g. it has been reconfigured)
+- ``EACCESS``  - Anothe resources is currently trying to access the port
```
Comment (Octogonapus) [inline v5/api/c/vision.rst]:
```suggestion
- ``EACCESS``  - Another resources is currently trying to access the port
```

File: v5/api/cpp/vision.rst [hunk 3]
```diff
@@ -647,9 +647,9 @@ Reads up to object_count object descriptors into object_arr.
 This function uses the following values of errno when an error state is
 reached:
 
+- ``ENXIO``  -The given value is not within the range of V5 ports (1-21), or fewer than object_count number of objects were found.
```
Comment (Octogonapus) [inline v5/api/cpp/vision.rst]:
```suggestion
- ``ENXIO``  - The given value is not within the range of V5 ports (1-21), or fewer than object_count number of objects were found.
```

File: v5/api/c/motors.rst (hunk 1 above)
Comment (HotelCalifornia) [reply v5/api/c/motors.rst]: https://github.com/purduesigbots/pros/blob/ab99a3391c8d88129e479ff6ce450afe7e569d10/include/pros/motors.h#L78

File: v5/api/c/vision.rst (hunk 2 above)
Comment (HotelCalifornia) [reply v5/api/c/vision.rst]: https://github.com/purduesigbots/pros/blob/ab99a3391c8d88129e479ff6ce450afe7e569d10/include/pros/vision.h#L539 but yeah this should be fixed either way

File: v5/api/c/motors.rst (hunk 1 above)
Comment (baylessj) [reply v5/api/c/motors.rst]: Yeah, I found out last season that it's always in RPM regardless of the motor's encoder units. Seemed a bit strange but held up in my testing.

**Pull request merged**